dev-packages/e2e-tests/lib/getTestMatrix.ts

```diff
@@ -48,10 +48,24 @@ function run(): void {
 
   const { base, head = 'HEAD', optional } = values;
 
+  // For GitHub Action debugging
+  // eslint-disable-next-line no-console
+  console.error(`Parsed command line arguments: base=${base}, head=${head}, optional=${optional}`);
+
   const testApplications = globSync('*/package.json', {
     cwd: `${__dirname}/../test-applications`,
   }).map(filePath => dirname(filePath));
 
+  // For GitHub Action debugging (using stderr the 'matrix=...' output is not polluted)
+  // eslint-disable-next-line no-console
+  console.error(
+    `Discovered ${testApplications.length} test applications${
+      testApplications.length > 0
```

Comment:
let's just make this `testApplications.join(', ')`? Doesn't really matter if this list is long I suppose 😅